sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -105,6 +105,21 @@ extensions:
       timeout: 7200
       repository: "aos-cd-jobs"
       script: |-
+        crio_tag="$( cat ./ORIGIN_PKG_VERSION | cut -d'-' -f2 | cut -d'.' -f1,2 )"
```

Comment:
I was also interested in _this_ being fragile. Can you use regex in the `case` statements?

Reply:
You mean extracting something like "3.9" with a regexp from that file? I could do that yeah

Reply:
Right

Reply:
fixed, now using grep